prep-javascript-objects/main.js

```diff
@@ -0,0 +1,19 @@
+var person = {
+  firstName: 'John',
+  lastName: 'Smith',
+  hobby: 'baseball',
+  job: 'plumber',
+  previousJob: 'teacher'
+};
+console.log(person);
+var fullName = 'John ' + 'Smith';
+var nameDeclare = "The person's full name is";
+console.log(nameDeclare, fullName);
+var currentJob = 'plumber';
+var currentJobDeclare = "The person's current job is";
+console.log(currentJobDeclare, currentJob);
+var previousJob = 'teacher';
+var previousJobDeclare = "The person's previous job was";
+console.log(previousJobDeclare, previousJob);
```

Comment:
Down in this area is where you would use dot notation to set the `job` and `previousJob` properties on the `person` object.

Reply:
done!
<img width="1440" alt="Screen Shot 2021-09-22 at 4 07 47 PM" src="https://user-images.githubusercontent.com/88061673/134414626-38cbbdfe-4328-4ede-957c-b9df39a45065.png">